conf/.env

```diff
@@ -84,7 +84,7 @@ NODEINFO=true
 WEBFINGER=true
 
 ## Mail Configuration (Post-Installer)
-MAIL_DRIVER=smtp
+MAIL_DRIVER=sendmail
 MAIL_HOST=localhost
 MAIL_PORT=25
 MAIL_USERNAME=null
```

Comment:
Eeeh by the way, a new mechanism landed in YunoHost 11.2 a couple weeks ago which now allows app to send proper, DKIM-signed mails assuming you do configure the mail username and password ... cf the `allow_email` property in the system user resource : https://yunohost.org/en/contribute/packaging_apps/resources/appresources#system-user

Dunno if that helps but what I do know is that, wether you're using smtp or sendmail, as long as you are not properly authenticated with a login/passwords, emails wont be DKIM-signed and therefore likely to be marked as spam

Reply:
Thanks for the reminder.

What do we need to use it ?
The mail password and mail username (username@domain.tld I guess ?) ?

Reply:
@alexAubin yes, we tried this in another PR #229 
the mail user was created, but somehow there were no mails sent.

does maybe something pop in your eye here?
```
## Mail Configuration (Post-Installer)
MAIL_DRIVER=smtp
MAIL_HOST=127.0.0.1
MAIL_PORT=587
MAIL_USERNAME=__APP__
MAIL_PASSWORD=__MAIL_PWD__
MAIL_ENCRYPTION=null
MAIL_FROM_ADDRESS="__APP__@__DOMAIN__"
MAIL_FROM_NAME="Pixelfed"
```

Reply:
I'll try on a sub-PR.